gentest/fixtures/YGAlignContentTest.html

```diff
@@ -29,3 +29,23 @@
   <div style="width: 50px;"></div>
   <div style="width: 50px;"></div>
 </div>
+
+<div id="align_content_stretch_row" style="width: 150px; height: 100px; flex-wrap: wrap; flex-direction: row; align-content: stretch;">
+  <div style="width: 50px;">
+    <div style="flex: 1"></div>
```

Comment:
I want to test if the child is correctly resized, when the parent size is changed via ```align-content:stretch```